### ISSUE TITLE
BybitClient.CopyTradingApi.Trading.PlaceOrderAsync always returns error

### ISSUE THREAD

**Issue by ThaiCat** [JKorf/Bybit.Net#70]
I tried different OrderType, Different OrderSide, quantity, manually set price, and 2 different currency pairs. Result is the same

{130076: params invalid }

I did not find this error code in the docs ether.

**Comment by kulikov-dev** [comment]
@ThaiCat Can you send the part of code where you form and send your order?

**Comment by ThaiCat** [comment]
code:
```

                decimal price_ = 0.45m;

                var orderResponce = await _BybitClient.CopyTradingApi.Trading.PlaceOrderAsync(
                      symbol: "BTCUSDT",
                      side: Bybit.Net.Enums.OrderSide.Buy,// orderSide,
                      type: Bybit.Net.Enums.OrderType.Limit,
                      quantity: 0.001m,
                      price: 19000
                );
                Log(orderResponce.Error?.ToString());

                orderResponce = await _BybitClient.CopyTradingApi.Trading.PlaceOrderAsync(
                      symbol: "XRPUSDT",
                      side: Bybit.Net.Enums.OrderSide.Buy,
                      type: Bybit.Net.Enums.OrderType.Limit,
                      quantity: 1,
                      price: price_                                   
                );
                Log(orderResponce.Error?.ToString());

                orderResponce = await _BybitClient.CopyTradingApi.Trading.PlaceOrderAsync(
                      symbol: "XRPUSDT",
                      side: Bybit.Net.Enums.OrderSide.Sell,
                      type: Bybit.Net.Enums.OrderType.Market,
                      quantity: 1,
                      price: price_
                );
                Log(orderResponce.Error?.ToString());

                orderResponce = await _BybitClient.CopyTradingApi.Trading.PlaceOrderAsync(
                      symbol: "XRPUSDT",
                      side: Bybit.Net.Enums.OrderSide.Buy,
                      type: Bybit.Net.Enums.OrderType.Limit,
                      quantity: 1,
                      price: price_
                );
                Log(orderResponce.Error?.ToString());

                orderResponce = await _BybitClient.CopyTradingApi.Trading.PlaceOrderAsync(
                      symbol: "XRPUSDT",
                      side: Bybit.Net.Enums.OrderSide.Buy,
                      type: Bybit.Net.Enums.OrderType.Limit,
                      quantity: 1,
                      price: price_
                );
                Log(orderResponce.Error?.ToString());

```

output
[19:39:00.1720] Bybit XRPUSDT 130076: params invalid 
[19:39:00.4892] Bybit XRPUSDT 130076: params invalid 
[19:39:00.8163] Bybit XRPUSDT 130076: params invalid 
[19:39:01.1273] Bybit XRPUSDT 130076: params invalid 
[19:39:01.4111] Bybit XRPUSDT 130076: params invalid

**Comment by JKorf** [comment]
Are you a principal trader? Currently the API is not available for followers

**Comment by ThaiCat** [comment]
yes, i am
![bb](https://user-images.githubusercontent.com/4564078/198751681-717f678e-a75c-4720-bff7-c97ab1b27ca2.png)

**Comment by JKorf** [comment]
I'm sorry, but I don't think I can help you with this as I'm not a principal trader myself. You can try the telegram api support channel https://t.me/BybitAPI

**Comment by OxfordGrid** [comment]
I'm getting the same error. @JKorf it is possible to sign up for copy trading on testnet in a matter of minutes. Would this help to debug?

**Comment by OxfordGrid** [comment]
Ok, I've found a bug that causes it. I'll create a PR later today.

**Comment by OxfordGrid** [comment]
@kulikov-dev Oh, excellent, thanks man! So now we just need to wait for the merge :)

**Comment by JKorf** [comment]
Has been merged and version 1.3.1 has been pushed
